developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md

```diff
@@ -5,67 +5,70 @@ id: "install-aptos-cli"
-1. Ensure you have `git` installed https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
-2. Clone the Aptos core repo:  `git clone https://github.com/aptos-labs/aptos-core.git`
-3. Change directory into `aptos-core` directory: `cd aptos-core`
-4. Run the dev setup script to prepare your environment: `./scripts/dev_setup.sh -yp`
-5. Source the profile file `source ~/.profile`
-
-This command should work on MacOS and Linux flavors like Ubuntu or CentOS. (Windows is currently not supported).
-
-Notice that you have to include environment variable definitions in `~/.profile` into your shell. Depending on your
-setup, the  `~/.profile` may be already automatically loaded for each login shell, or it may not. If not, you may
-need to add `. ~/.profile` to your `~/.bash_profile` or other shell configuration manually.
-
-6. You should now be able to prove an example
-```bash
-aptos move prove --package-dir aptos-move/move-examples/hello_prover/
-```
-
-## Install from Git
-### Step 1: Install from Git
-
-Start by cloning the `aptos-core` GitHub repo from [GitHub](https://github.com/aptos-labs/aptos-core).
-
-1. Ensure you have `git` installed https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
-2. Clone the Aptos core repo:  `git clone https://github.com/aptos-labs/aptos-core.git`
-3. Change directory into `aptos-core` directory: `cd aptos-core`
-4. Run the dev setup script to prepare your environment: `./scripts/dev_setup.sh`
-5. Update your current shell environment: `source ~/.cargo/env`
-6. Checkout the correct branch `git checkout --track origin/branch`, where branch is
-    - `devnet` for building on Devnet
-    - `testnet` for building on Testnet
-    - `main` for the current development branch
-7. Build the CLI tool: `cargo build --package aptos --release`
-8. The binary will be available in `target/release/aptos`
-9. (Optional) Move this executable to a place on your path e.g. `~/bin/aptos`
-
-### Step 2 (optional): Install the dependencies of Move Prover
-
-1. Run the dev setup script to prepare your environment: `./scripts/dev_setup.sh -yp`
-2. Source the profile file `source ~/.profile`
-
-This command should work on MacOS and Linux flavors like Ubuntu or CentOS. (Windows is currently not supported).
-
-Notice that you have to include environment variable definitions in `~/.profile` into your shell. Depending on your
-setup, the  `~/.profile` may be already automatically loaded for each login shell, or it may not. If not, you may
-need to add `. ~/.profile` to your `~/.bash_profile` or other shell configuration manually.
-
-3. You should now be able to prove an example
-```bash
-aptos move prove --package-dir aptos-move/move-examples/hello_prover/
-```
\ No newline at end of file
+The `aptos` tool is a command line interface (CLI) for developing on the Aptos blockchain, debugging, and for node operations. This document describes how to install the `aptos` CLI tool. See [Use Aptos CLI](use-aptos-cli) for how to use the CLI.
```

Comment:
probably say `for developing on the Aptos blockchain, for debugging move contracts, and for node operations` or
`for developing on the Aptos blockchain, debugging move contracts, and node operations`.

1. `move contracts` to help with what it debugs
2. `for` should either show up in all 3 or only at the beginning (as far as I remember from English class)